src/ios/CDVCamera.m

```diff
@@ -358,6 +358,7 @@ - (NSData*)processImage:(UIImage*)image info:(NSDictionary*)info options:(CDVPic
                 // use image unedited as requested , don't resize
                 data = UIImageJPEGRepresentation(image, 1.0);
             } else {
+                data = UIImageJPEGRepresentation(image, [options.quality floatValue] / 100.0f);
                 if (options.usesGeolocation) {
```

Comment:
When using the "unedited" image above, the geolocation option is ignored -- is that correct behavior? Or should the if (options.usesGeolocation) be moved to outside/after the else block?